SandWorm/Analytics/Contours.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using Rhino.Geometry;
+
+namespace SandWorm.Analytics
+{
+    public class Contours : Analysis.MeshGeometryAnalysis
+    {
+        public Contours() : base("Show Contour Lines")
+        {
+        }
+
+        public override void GetGeometryForAnalysis(ref List<GeometryBase> outputGeometry, int contourInterval, Mesh mesh)
+        {
+            var bounds = mesh.GetBoundingBox(false);
+            var originStart = new Point3d(0, 0, bounds.Min.Z);
+            var originEnd = new Point3d(0, 0, bounds.Max.Z);
+            var contours = Mesh.CreateContourCurves(mesh, originStart, originEnd, contourInterval);
```

Comment:
Nice! Looking forward to optimizing this one. I'm sure we can beat this algorithm by a fair margin, just as we did it with elevation & slope analysis.